### PR TITLE
SCRUM-5/S RAgregar ls métodos de assertion en su apartado correspondi…

### PR DESCRIPTION
**Pull request by AlisonGuzman2006**
Add the restructuring of the assertion methods of the endpoint test cases.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1 +1,10 @@
-BASE_URI = 'https://espo.spartan-soft.com/api/v1'
\ No newline at end of file
+BASE_URI = 'https://espo.spartan-soft.com/api/v1'
+USERNAME = "admin"
+PASSWORD = "admin"
+calls_params = {
```
Comment (Jasonn5) [inline config.py]: Use capital letters the same as in the other variables

File: conftest.py
Comment (Jasonn5) [inline conftest.py:1]: approved

File: src/assertions/assert_headers.py
Comment (Jasonn5) [inline src/assertions/assert_headers.py:1]: rename to assertion_headers.py

File: src/assertions/assertion_schemas.py
Comment (Jasonn5) [inline src/assertions/assertion_schemas.py:1]: approved

File: src/espocrm_api/api_request.py
Comment (Jasonn5) [inline src/espocrm_api/api_request.py:1]: approved

File: src/espocrm_api/calls_endpoints.py
Comment (Jasonn5) [inline src/espocrm_api/calls_endpoints.py:1]: approved

File: src/resources/schemas/get_completeFields_all_calls_schema.json
Comment (Jasonn5) [inline src/resources/schemas/get_completeFields_all_calls_schema.json:1]: approved

File: tests/calls/test_getCalls.py
Comment (Jasonn5) [inline tests/calls/test_getCalls.py:1]: approved

**Review by Jasonn5 — CHANGES_REQUESTED** [review]

File: config.py
Comment (Jasonn5) [inline config.py:1]: looks good

File: conftest.py
Comment (Jasonn5) [inline conftest.py:1]: Good contribution

**Review by Jasonn5 — APPROVED** [review]
Approved

File: config.py
Comment (Diego-Claros17) [reply config.py:1]: Appears acceptable

File: conftest.py
Comment (Diego-Claros17) [reply conftest.py:1]: Good work

File: src/espocrm_api/calls_endpoints.py
Comment (Diego-Claros17) [inline src/espocrm_api/calls_endpoints.py:1]: Nice work, good reestructuration

File: src/espocrm_api/api_request.py
Comment (Diego-Claros17) [inline src/espocrm_api/api_request.py:1]: Nice work

File: tests/calls/test_getCalls.py
Comment (Diego-Claros17) [inline tests/calls/test_getCalls.py:1]: All tests looks great and begin to follow the guidelines of the code restructuring

**Review by Diego-Claros17 — APPROVED** [review]
I have reviewed the changes and they do not generate any conflict and it works correctly, good job.

**Pull request merged**